File: ConsoleApp1/Sort/SelectionSort.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Sort
{
    internal class SelectionSort
    {
        static void Selection(int[] A)
        {
            int n = A.Length;
            for (int i = 0;  i < n-1; i++)
            {
                int i_min = i;
                int v_min = A[i];
                for (int j = i; j < n; j++)
                {
                    if (A[j] < v_min)
                    {
                        v_min = A[j];
                        i_min = j;
                    }
                }
                int temp = A[i];
                A[i] = A[i_min];
                A[i_min] = temp;
            }
        }
        static void Main(string[] args)
        {
            Console.Write("Nhập số lượng mảng: ");
            int n = Convert.ToInt32(Console.ReadLine());
            int[] A = new int[n];
            for (int i = 0; i < n; i++)
            {
                Console.Write($"Nhập giá trị A{i + 1}: ");
                A[i] = Convert.ToInt32(Console.ReadLine());
            }
            Selection(A);
            for (int i = 0; i < n; i++)
            {
                Console.Write(A[i] + " ");
            }
        }
    }
}

File: Sort_Python/New/Selection_Sort.py
def Sort(A, n):
    for i in range(0, n):
        # Lưu giá trị và vị trí nhỏ nhất 
        i_min = i
        v_min = A[i]
        for j in range(i, n): # Để bỏ qua các giá trị ở đầu đã sắp xếp
            if (A[j] < v_min):
                v_min = A[j]
                i_min = j
        # Đổi vị trí với i 
        A[i], A[i_min] = A[i_min], A[i]
A = [12, 234, 11, 5, 50, 200]
n = len(A)
Sort(A, n)
print(A)